ModKit/UI/UI+Browser.cs

```diff
@@ -85,11 +86,12 @@ public class Browser<Definition, Item> : Browser {
             public bool availableIsStatic { get; private set; }
             private List<Definition> _availableCache;
             public void OnShowGUI() => needsReloadData = true;
-            public Browser(bool searchAsYouType = true, bool availableIsStatic = false, bool isDetailBrowser = false) {
+            public Browser(bool searchAsYouType = true, bool availableIsStatic = false, bool isDetailBrowser = false, bool shouldLocalize = false) {
```

Comment:
I think we should just always localize on all of our internal UI strings.  Let the dev choose to localize external ones.  Let's ship a starter localization file in mod kit sources